pkg/scheduler/plugins/util/k8s/snapshot.go

```diff
@@ -51,8 +51,7 @@ func NewEmptySnapshot() *Snapshot {
 }
 
 // NewSnapshot initializes a Snapshot struct and returns it.
-func NewSnapshot(pods []*v1.Pod, nodes []*v1.Node) *Snapshot {
-	nodeInfoMap := createNodeInfoMap(pods, nodes)
```

Comment:
The root cause is that original `nodeInfoMap` was not updated in one scheduling cycle, right?